.github/workflows/performance-threshold-update.yml

```
@@ -0,0 +1,215 @@
+            ---
+            *Generated by Performance Threshold Automation*
+          branch: perf-threshold-update-${{ github.run_number }}
+          delete-branch: true
+          labels: |
+            type/maintenance
+            area/performance
+            automated
+      
+      - name: Summary
+        if: always()
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:10:58: Double quote to prevent globbing and word splitting

(shellcheck)